src/OrchardCore.Modules/OrchardCore.Facebook/Filters/FBInitFilter.cs

```diff
@@ -46,16 +46,17 @@ public async Task OnResultExecutionAsync(ResultExecutingContext context, ResultE
                             options = string.Concat(options, ",", settings.FBInitParams, "}");
                         }
                         var require = new RequireSettings() { Name = "fb", Type = "script", };
-                        var resource=_resourceManager.FindResource(require);
+                        var resource = _resourceManager.FindResource(require);
 
                         _resourceManager.RegisterResource("script", "fb");
 
-                        _resourceManager.RegisterFootScript(new HtmlString(@"<script src=""/OrchardCore.Facebook/sdk/fb.js"" type=""text/javascript""></script>"));
-                        _resourceManager.RegisterFootScript(new HtmlString(@"<script src=""/OrchardCore.Facebook/sdk/fbsdk.js"" type=""text/javascript""></script>"));
+                        var pathBase = context.HttpContext.Request.PathBase.ToString();
+                        _resourceManager.RegisterFootScript(new HtmlString($"<script src=\"{pathBase}/OrchardCore.Facebook/sdk/fb.js\" type=\"text/javascript\"></script>"));
```

Comment:
Isn't there a way to to the same API call that the tag helper is doing? I assume the tag helper is not creating a tag directly but just injecting a new resource descriptor.

Reply:
Okay i will try this